packages/yoroi-extension/app/api/ada/index.js

```diff
@@ -1431,6 +1451,12 @@ export default class AdaApi {
           neededHashes: new Set(),
           wits: new Set(),
         },
+        trezorTCatalystRegistrationTxSignData:
+          request.trezorTWallet &&
+            {
+              publicVotingKey: request.trezorTWallet.publicVotingKey,
+              nonce: request.absSlotNumber,
+            },
```

Comment:
@yushih , you are using `.flag && value` here, which I think might leave the `trezorTCatalystRegistrationTxSignData` to have value `false` or something like that, in cases when the flag is not truthy. Let's use
```
request.trezorTWallet ? { ... } : undefined,
```

Because in some other place eventually someone might use a `!= null` check and then we'll have a problem.